index.html

```diff
@@ -319,6 +318,27 @@ <h2 class="about-title">About<br />Myself<br /></h2>
         <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
       </div>
     </section>
+    <section class="contact" id="contact">
+      <div class="inner-div">
+        <h2>Contact me</h2>
+        <p id="inner-div">If you have an application you are interested in developing, a feature that you need
+          built or a project that needs coding. I'd love to help with it </p>
+      </div>
+      <form action="https://formspree.io/f/mvoljlod" id="form" class="form" method="post">
+        <ul class="inner-list">
+          <li><input type="text" class="form-control" placeholder="Full name" name="name" id="name" size="40" maxlength="30">
+          </li>
+          <li> </li>
+          <li><input type="email" class="form-control" name="email" id="email" placeholder="Email address">
+          </li>
+          <li> </li>
+          <li><textarea name="text" class="form-control" id="text" placeholder="Message" maxlength="500"></textarea></li>
+          <li> </li>
+          <li class="flex-item6" id="submit-btn"><button type="submit">Get In Touch</button></li>
+          <li class="shape"><img src="./images/Shape.png" alt="shape"></li>
```

Comment:
- Optional
You don't need to make the form elements as the list items wrapped in the list. By default, the form will behave as a list but not exactly as a list since list elements behave as block-level elements and form elements behave as inline elements. However, that said, it is completely useless to put wrap them in the list.